freebridge-ai/routers/recommendation.py

```
@@ -28,36 +31,114 @@ def get_vs():
+        
+        if data.get('type') == "experience":
+            prefix = "exp"
+        elif data.get('type') == "job_posting":
+            prefix = "job"
+        else:
+            prefix = "user"
+            
+        vs.add_documents([doc], ids=[f"{prefix}:{data['id']}"])
+        
+        logger.info(f"Sync Success: {prefix}:{data['id']} | Status: {data.get('status')}")
+        return {"success": True}
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)